database/migrations/settings/2024_05_07_104300_create_customer_experience_faqs_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+
+class CreateCustomerExperienceFaqsTable extends Migration
+{
+
+    public function up(): void
+    {
+        Schema::create('customer_experience_faqs', function(Blueprint $table){
+            $table->increments('id');
+            $table->string('faq_question');
+            $table->longText('faq_answer');
```

Comment:
You can omit the `faq_` part of the column name. It's already inferred by the table name.

```suggestion
            $table->string('question');
            $table->longText('answer');
```

Just remember to also update the rest of the codebase with a find and replace.